src/node/test/async_test.js

```diff
@@ -36,7 +36,7 @@
 var assert = require('assert');
 
 var grpc = require('..');
-var math = grpc.load(__dirname + '/../../proto/math/math.proto').math;
+var math = grpc.load(__dirname + '/../../proto/math/math.proto').lookup('math');
```

Comment:
I would like to keep at least one test file that doesn't use `lookup`, to ensure that uses of the current API will still work. I think this file is a good one to do that in.